lib/msal-core/test/UserAgentApplication.spec.ts

```diff
@@ -827,8 +827,43 @@ describe("UserAgentApplication.ts Class", function () {
                 console.error("Error in assertion: " + JSON.stringify(err));
             });
         });
+
+        it("tests getCachedToken is skipped when force is set true", function (done) {
```

Comment:
Do you need another test for when `forceRefresh: false`?

Reply:
+1

Reply:
I don't think so, since it defaults to false and all other tests don't pass in.  The check is falsey

Reply:
Default case is the normal case for ATS, so we should have some testing for that already.